students/ColeP/.idea/vcs.xml

```diff
@@ -0,0 +1,6 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
When committing files, you do not need to commit changes that come from your IDE. Usually this can make things messy, I would recommend taking this out (and not including it in future Pull Requests)